cmd/zoekt-sourcegraph-indexserver/queue.go

```diff
@@ -82,17 +82,22 @@ func NewQueue(backoffDuration, maxBackoffDuration time.Duration, l sglog.Logger)
 	return q
 }
 
-// Pop returns the opts of the next repo to index. If the queue is empty ok is
-// false.
-func (q *Queue) Pop() (opts IndexOptions, ok bool) {
+type QueueItem struct {
```

Comment:
Not an ideal name since we already have `queueItem` 🤔

Reply:
looks good to me. queueItem has the private info, and QueueItem the info for users of the API